packages/node/src/algorand/utils.algorand.ts

```diff
@@ -79,6 +80,26 @@ export function filterBlockModulo(
   return block.round % modulo === 0;
 }
 
+function txComparator(a: any, b: any): boolean {
+  if (Array.isArray(a)) {
+    for (let i = 0; i < a.length; i++) {
+      const valA = a[i];
+      const valB = b[i];
+
+      if (valA === null) {
```

Comment:
here we assume that valA is from filter and valB is from chain data. should we name it accordingly?